packages/api/src/external/ehr/athenahealth/command/get-patient.ts

```diff
@@ -115,7 +140,33 @@ function createMetriportPatientDemos(patient: PatientResource): PatientDemoData[
       });
     });
   });
-  return patientPatients;
+  return patientDemoFilters;
+}
+
+function createMetriportPatientDemo(
+  patient: PatientResource
+): Omit<PatientCreateCmd, "cxId" | "facilityId"> {
+  const firstNames: string[] = [];
+  const lastNames: string[] = [];
+  const addressArray = createMetriportAddresses(patient);
+  const contactArray = createMetriportContacts(patient);
+  patient.name.map(name => {
+    const lastName = name.family;
+    if (lastName === "") return;
+    lastNames.push(toTitleCase(lastName));
+    name.given.map(firstName => {
+      if (firstName === "") return;
+      firstNames.push(toTitleCase(firstName));
+    });
+  });
```

Comment:
Pretty similar to `createMetriportPatientDemoFilters`, worth its own function so we only define the mapping logic for names in one place?

Reply:
Done.